components/welcome/Welcome.tsx

```diff
@@ -0,0 +1,18 @@
+import { ImageBackground, ImageSourcePropType, Text } from 'react-native'
+
+interface Props {
+  title: string
+  detail: string
+  url: ImageSourcePropType
```

Comment:
usar un nombre mas declarativo para la prop de un tipo especifico, en su lugar : 
```suggestion
  imageSource: ImageSourcePropType
```